### DISCUSSION TITLE
librime里引用的opencc在Windows下不能正确解析出用户词库目录包含中文

### DISCUSSION THREAD

**Discussion by Techince** [rime/home#1294] [General]
在Windows下使用小狼毫时发现，如果用户词库目录所在路径中包含中文，那么opencc的字典设置的文件名会解析不正确，它是按utf8解析的，而传入的格式是多字节字符集，希望作者修改一下。

**Comment by yanhuacuo** [comment]
opencc 不支持中文路径

**Comment by yanhuacuo** [comment]
不要使用中文路径

**Reply by Techince** [reply]
Opencc支持中文路径的，我自己修改过代码了。不过还是想通知作者修改一下，方便日后升级库时不分裂。
